internal/runner/runner.go

```
@@ -624,6 +686,12 @@ func (r *Runner) RunEnumeration() error {
 		Retries:             r.options.Retries,
 	}, "")
 
+	if r.dastServer != nil {
+		if err := r.dastServer.Start(); err != nil {
+			r.dastServer.Start()
```

golangci-lint (1.62.2)

Error return value of `r.dastServer.Start` is not checked

(errcheck)